backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/material/interceptor/transactionevent/TransactionEventFactory.java

```diff
@@ -233,56 +231,51 @@ else if (transaction.getInventoryLineId() != null)
 		return events.build();
 	}
 
-	private List<MaterialEvent> createEventForInventoryLine(
+	@NonNull
+	private MaterialEvent createEventForInventoryLine(
 			@NonNull final TransactionDescriptor transaction,
 			final boolean deleted)
 	{
 		final boolean directMovementWarehouse = isDirectMovementWarehouse(transaction.getWarehouseId());
 
 		final I_M_InventoryLine inventoryLineRecord = inventoryDAO.getLineById(transaction.getInventoryLineId());
 
-		final List<HUDescriptor> huDescriptors = huDescriptorsViaInventoryLineService.createHuDescriptorsForInventoryLine(inventoryLineRecord, deleted);
+		final BigDecimal deltaQty = inventoryLineRecord.getQtyCount().subtract(inventoryLineRecord.getQtyBook());
```

Comment:
i think there is nothing wrong with *also* keeping providing the huDescriptors to the event..in case some event cosumers might want to use them

Reply:
huDescriptors  added